apis/nucleus/src/components/listbox/utils/deduce-frequency-mode.js

```diff
@@ -0,0 +1,5 @@
+export default function deduceFrequencyMode(pages) {
+  const hasPercentSign = pages.some(({ qMatrix }) => qMatrix.some(([item]) => !!item.qFrequency?.match('%')));
+  const OTHER = '-'; // treat all other as '-' (change later if we need to identify other modes)
+  return hasPercentSign ? 'P' : OTHER;
```

Comment:
this could be wrong if all values in pages has no frequency as they are excluded values.
is that OK? out could that cause problems with scrolling?

Reply:
🤷 Like I said before, the only workaround I can think of would be to get frequency mode from properties. Should we do that instead?
At the same time, it is more of a **visual** bug than a **functional** bug. I think it should be ok given that it is an edge case and will go away once we have frequency glyph count in the Engine response.

Reply:
It can also be seen when scrolling, but I agree that it is "more of a visual bug than a functional bug"
and we can probably live with it.